casper/src/main/scala/coop/rchain/casper/util/comm/CommUtil.scala

```diff
@@ -29,6 +29,8 @@ trait CommUtil[F[_]] {
   // Broadcast packet in chunks (stream)
   def streamToPeers(packet: Packet): F[Unit]
 
+  def sendToPeer(message: Packet, peer: PeerNode): F[Unit]
```

Comment:
https://github.com/rchain/rchain/blob/8a69a559f1d4c96761251a81c974b74883e2ccd9/comm/src/main/scala/coop/rchain/comm/transport/TransportLayerSyntax.scala#L22-L27
This is already implemented as TransportLayer syntax